docs/fsharp/get-started/get-started-vscode.md

```diff
@@ -46,51 +46,37 @@ You can install Visual Studio Code from the [code.visualstudio.com](https://code
 
 Next, click the Extensions icon and search for "Ionide":
 
-The only plugin required for F# support in Visual Studio Code is [Ionide-fsharp](https://marketplace.visualstudio.com/items?itemName=Ionide.Ionide-fsharp). However, you can also install [Ionide-FAKE](https://marketplace.visualstudio.com/items?itemName=Ionide.Ionide-FAKE) and to get [FAKE](https://fsharp.github.io/FAKE/) support and [Ionide-Paket](https://marketplace.visualstudio.com/items?itemName=Ionide.Ionide-Paket) to get [Paket](https://fsprojects.github.io/Paket/) support. FAKE and Paket are additonal F# community tools for building projects and managing dependencies, respectively.
+The only plugin required for F# support in Visual Studio Code is [Ionide-fsharp](https://marketplace.visualstudio.com/items?itemName=Ionide.Ionide-fsharp). However, you can also install [Ionide-FAKE](https://marketplace.visualstudio.com/items?itemName=Ionide.Ionide-FAKE) to get [FAKE](https://fsharp.github.io/FAKE/) support and [Ionide-Paket](https://marketplace.visualstudio.com/items?itemName=Ionide.Ionide-Paket) to get [Paket](https://fsprojects.github.io/Paket/) support. FAKE and Paket are additional F# community tools for building projects and managing dependencies, respectively.
 
 ## Creating your first project with Ionide
 
 To create a new F# project, open Visual Studio Code in a new folder (you can name it whatever you like).
 
-### [Windows](#tab/windows)
-
-Next, open the Command Palette (Ctrl+Shift+P) and type the following:
-
->F# new project
-
-### [macOS](#tab/macos)
-
-Next, open the Command Palette (⌘+Shift+P) and type the following:
-
->F# new project
-
-### [Linux](#tab/linux)
-
-Next, open the Command Palette (Ctrl+Shift+P) and type the following:
-
->F# new project
+Next, open the command pallette (**View > Command Pallette) and type the following:
```

Comment:
Close bold formatting: `(**View > Command Pallette) and...` should be `(**View > Command Pallette**) and...`